charts/metallb/values.yaml

```diff
@@ -0,0 +1,342 @@
+# Default values for metallb.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+loadBalancerClass: ""
```

Comment:
An explanation here wouldn't hurt

Reply:
This is again smth that is relevant for the upstream as well and can be addressed there